### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by andriy-shymkiv**

### CODE REVIEW

**Review by taraselnik — APPROVED** [review]
perfect, but how? )) give me a link please to read about

**Comment by Andriy-Shymkiv** [comment]
> perfect, but how? )) give me a link please to read about

krutyv vertyv i poluchylos'

File: src/index.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+</head>
+<body>
+
+<script src="./inverseRobot.js"></script>
+</body>
+</html>
```
Comment (mykola-koval) [inline src/index.html:13]: why do we need this?

File: src/inverseRobot.js [hunk 2]
```diff
@@ -24,7 +24,16 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const inverse = {};
+
+  for (const key in robot) {
+    if (inverse[robot[key]]) {
```
Comment (mykola-koval) [inline src/inverseRobot.js:30]: checklist 2

**Review by mykola-koval — CHANGES_REQUESTED** [review]

**Review by mykola-koval — APPROVED** [review]

**Pull request closed without merging**